src/appleseed/renderer/modeling/bssrdf/bssrdffactoryregistrar.h

```diff
@@ -36,10 +36,16 @@
 #include "foundation/utility/api/apiarray.h"
 #include "foundation/utility/autoreleaseptr.h"
 #include "foundation/utility/searchpaths.h"
+#include "foundation/platform/sharedlibrary.h"
 
 // appleseed.main headers.
 #include "main/dllsymbol.h"
 
+//standard headers
+#include <string>
```

Comment:
It doesn't look like `<string>` and `<vector>` are required, please remove them. Moreover you're including `<memory>` which makes sense but then you're using `boost::shared_ptr<>` which is completely unrelated (and for that you're missing the header). Switch to `std::shared_ptr<>`.

Reply:
I was searching for the difference between boost::shared_ptr<> and std::shared_ptr<> , thanks I will!